docs/getting-started/verification.md

```diff
@@ -0,0 +1,211 @@
+* Serial communication devices, refer to [your target
+  page](../supported-targets/targets.md) and check how to establish serial
+  communication with your target;
+* A cable for ethernet connection with your target.
+
+## Verification
+
+The verification is an execution of a set of basic commands on the running
+system to prove the functionality of Zarhus OS.
+
+### Booting platform
+
+This is the basic functionality of all OS'es. To verify this you need to connect
+to the serial port of your target and start communication with your host device,
+check [your target page](../supported-targets/targets.md) on how to do so.
+
+Plug in target power supply after the connection has been established and check
+for following logs:
+
+```bash
+(...)
+
+Starting kernel ...
+
+[    0.000000] Booting Linux on physical CPU 0x0000000000 [0x412fd050]
+[    0.000000] Linux version 6.6.23-yocto-standard-00118-g2d01bc1d4eea (oe-user@oe-host) (aarch64-zarhus-linux-gcc (G4
+[    0.000000] KASLR disabled due to lack of seed
+[    0.000000] Machine model: Radxa Compute Module 3(CM3) IO Board
+[    0.000000] efi: UEFI not found.
+[    0.000000] earlycon: uart0 at MMIO32 0x00000000fe660000 (options '1500000n8')
+[    0.000000] printk: bootconsole [uart0] enabled
+[    0.000000] NUMA: No NUMA configuration found
+[    0.000000] NUMA: Faking a node at [mem 0x0000000000200000-0x00000000efffffff]
+[    0.000000] NUMA: NODE_DATA [mem 0xef8239c0-0xef825fff]
+[    0.000000] Zone ranges:
+[    0.000000]   DMA      [mem 0x0000000000200000-0x00000000efffffff]
+[    0.000000]   DMA32    empty
+[    0.000000]   Normal   empty
+[    0.000000] Movable zone start for each node
+[    0.000000] Early memory node ranges
+[    0.000000]   node   0: [mem 0x0000000000200000-0x00000000efffffff]
+[    0.000000] Initmem setup node 0 [mem 0x0000000000200000-0x00000000efffffff]
+[    0.000000] On node 0, zone DMA: 512 pages in unavailable ranges
+[    0.000000] cma: Reserved 32 MiB at 0x00000000ed800000 on node -1
+[    0.000000] psci: probing for conduit method from DT.
+[    0.000000] psci: PSCIv1.1 detected in firmware.
+[    0.000000] psci: Using standard PSCI v0.2 function IDs
+
+
+(...)
+[    0.950871] systemd[1]: systemd 255.4^ running in system mode (-PAM -AUDIT -SELINUX -APPARMOR +IMA -SMACK +SECCOMP)
+[    0.953829] systemd[1]: Detected architecture arm64.
+
+Welcome to Distro for Zarhus product 0.1.0 (scarthgap)!
+
+(...)
+[  OK  ] Reached target Bluetooth Support.
+[  OK  ] Reached target Multi-User System.
+         Starting Record Runlevel Change in UTMP...
+[  OK  ] Finished Record Runlevel Change in UTMP.
+[  OK  ] Finished Virtual Console Setup.
+
+Distro for Zarhus product 0.1.0 zarhus-machine-cm3 ttyS2
+
+zarhus-machine-cm3 login:
+```
+
+!!! note
+    Above logs are for Zarhus OS port for a Radxa board, but you should see
+    similar logs, because these are considered to be the common Linux boot logs.
+
+### Basic packages
+
```

Comment:
@DaniilKl I'm thinking about adding note that this is the verification procedure for product XYZ. What do you think? It would be much faster to find that way. XYZ should be the product ID.

Reply:
@artur-rs, what for? I understand that we base some of our commercial projects on Zarhus, but i do not see any reasons to add here product IDs. Why not to link internal documentation to the chapters of Zarhus documentation, and invest more time to Zarhus documentation? IMHO, additionally for all other users some IDs in documentation will look confusing and weird.

Reply:
@DaniilKl Ideally, we would like to avoid using internal docs, but this is a long-term topic. Product IDs could be added as a non-visible header, so the search engine should still help us find target docs by the PID from the search bar. **Note:** this thread is non-blocking in the PR review/merge

Reply:
https://github.com/zarhus/zarhus-issues/issues/16